app/src/styles.sass

```diff
@@ -36,6 +36,8 @@ input
     font-size: $fontsize
     margin-left: $medium-space
     padding: 0.4em 1em
+    min-width: 55%
+    max-width: 100%
```

Comment:
You're enforcing the width of all buttons! See for instance:
![image](https://user-images.githubusercontent.com/1835665/119457470-01c04f80-bd3c-11eb-8d0c-3f6726d31cf9.png)

Reply:
yes, I discovered it now too. I will modify it.